packages/dev/serializers/test/integration/glTFSerializer.test.ts

```diff
@@ -555,5 +554,92 @@ describe("Babylon glTF Serializer", () => {
                 expect(node.mesh).toEqual(0);
             }
         });
+        it("should not export a root conversion node", async () => {
+            const assertionData = await page.evaluate(() => {
+                return BABYLON.SceneLoader.AppendAsync("https://models.babylonjs.com/Tests/TwoQuads/", "TwoQuads.gltf", window.scene).then((scene) => {
+                    scene.getMeshByName("__root__")!.name = "renamedRoot";
+                    return BABYLON.GLTF2Export.GLTFAsync(window.scene!, "test").then((glTFData) => {
+                        const jsonString = glTFData.glTFFiles["test.gltf"] as string;
+                        const jsonData = JSON.parse(jsonString);
+                        return jsonData;
+                    });
+                });
+            });
+            expect(assertionData.nodes).toHaveLength(2);
+            expect(assertionData.scenes).toHaveLength(1);
+            expect(assertionData.scenes[0].nodes).toHaveLength(2);
+        });
+        it("should not duplicate a shared texture between materials", async () => {
+            const assertionData = await page.evaluate(() => {
+                const texture = new BABYLON.Texture("https://assets.babylonjs.com/environments/backgroundGround.png", window.scene!);
+                for (let i = 0; i < 2; i++) {
+                    const material = new BABYLON.PBRMaterial("mat" + i, window.scene!);
+                    material.bumpTexture = texture;
+                    BABYLON.MeshBuilder.CreateBox("box" + i).material = material;
+                }
+                return BABYLON.GLTF2Export.GLTFAsync(window.scene!, "test").then((glTFData) => {
+                    const jsonString = glTFData.glTFFiles["test.gltf"] as string;
+                    const jsonData = JSON.parse(jsonString);
+                    return jsonData;
+                });
+            });
+            expect(assertionData.textures).toHaveLength(1);
+            expect(assertionData.images).toHaveLength(1);
+        });
+        it("should not convert right-handed node transformations", async () => {
+            const transformsRH = {
+                translation: [0.2, 0.3, 0.4],
+                scale: [0.5, 0.6, 0.7],
+                rotation: [0.5, 0.5, 0.5, 0.5],

```

Comment:
These values don't break anything, but they are not valid Quaternion values. You might want to do something like this instead.
```suggestion
                rotation: Quaternion.FromEulerAngles(0, Math.PI, 0).asArray(),
```

Reply:
What do you mean by valid here? This is the quaternion from euler angles (0, PI/2, PI/2). 
I'm going to skip this, mainly because I don't want to import `Quaternion` in the test scope.

Reply:
Ok, I stand corrected, lol. I didn't expect the Quaternion to be such perfect numbers.